src/ImageSharp/Formats/Gif/GifDecoderCore.cs

```diff
@@ -363,8 +380,6 @@ private unsafe void ReadFrameColors(byte[] indices, byte[] colorTable, int color
 
             if (this.previousFrame == null)
             {
-                this.metaData.Quality = colorTableLength / 3;
```

Comment:
👍 Quality should be a property of the encoder. Having it global just confuses everybody.